File: Easy/26-remove-duplicates-from-sorted-array/try-9.py
/*
 * Problem #26: Remove Duplicates from Sorted Array
 * Difficulty: Easy
 * Submission: Try 9
 * status: Accepted
 * Language: python3
 * Date: 30/11/2024, 12:24:20
 * Link: https://leetcode.com/problems/remove-duplicates-from-sorted-array/
 */

class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        j=0
        for i in range(len(nums)):
            if(nums[j]!=nums[i]):
                j+=1
                nums[j]=nums[i]
        return j+1

        

File: Easy/242-valid-anagram/try-9.cpp
/*
 * Problem #242: Valid Anagram
 * Difficulty: Easy
 * Submission: Try 9
 * status: Accepted
 * Language: cpp
 * Date: 06/12/2024, 15:48:34
 * Link: https://leetcode.com/problems/valid-anagram/
 */

#include<algorithm>

class Solution {
public:
    bool isAnagram(string s, string t) {
        
        sort(t.begin(),t.end());
        sort(s.begin(),s.end());
        return s==t;    
    }
        
};
